src/odemis/driver/_mccdaq.py

```diff
@@ -0,0 +1,700 @@
+        is bitwise configurable, it may be any combination of inputs and outputs.
+        The return value will be the value seen at the port pins.
+
+         port_number:    AUXPORT    =   0x10
+                         Port A     =   0x01
+                         Port B     =   0x04
+        Bug: need leading 0 in command string.
+        '''
+
+        self.h.write([0x0, self.DIN, port_number, 0, 0, 0, 0, 0, 0])
+        try:
+            value = self.h.read(8, 500)
+        except:
+            print('DIn: error in reading.')
+        return (value[0])
+
+    def DOut(self, port_number, value):
+        '''
+        This command writes data to the DIO port bits that are configured as outputs.
+
+         port_number:    AUXPORT    =   0x10
+                         Port A     =   0x01
+                         Port B     =   0x04
+         value:          value to write to the port
+        '''
+        self.h.write([self.DOUT, port_number, value, 0, 0, 0, 0, 0])
+
+    def DBitIn(self, port_number, bit):
+        '''
+        This command reads an individual digital port bit.  It will return the value
+        seen at the port pin, so may be used for an input or output bit.
+
+         port_number:    AUXPORT    =   0x10
+                         Port A     =   0x01
+                         Port B     =   0x04
+
+         bit:            The bit to read (0-7)
+        '''
+
+        self.h.write([self.DBIT_IN, port_number, bit, 0, 0, 0, 0, 0])
+        try:
+            value = self.h.read(1, 500)
+        except:
+            print('DBitIn: error in reading.')
+        return (value[0])
+
+    def DBitOut(self, port_number, bit, value):
+        '''
+        This command writes an individual digital port bit.
+
+         port_number:    AUXPORT    =   0x10
+                         Port A     =   0x01
+                         Port B     =   0x04
+
+         bit:            The bit to read (0-7)
+         value:          The value to write to the bit (0 or 1)
+        '''
+        self.h.write([self.DBIT_OUT, port_number, bit, value, 0, 0, 0, 0])
+
+    #################################
+    #    Analog Input  Commands     #
+    #################################
+
+    def AIn(self, channel, gain):
+        '''
+        This command reads the value from an analog input channel, setting the desired
+        gain range first.  Note that single-ended inputs include 11 bits of resolution
+        while differential inputs have 12 bits of resolution.  Also, the 'range' code
+        used includes single-end/differential input setting.
+
+           channel: the channel to read (0-3 differential, or 0-7 single ended)
+           range:   the gain and input mode setting
+        '''
+
+        if (gain == self.SE_10_00V):
+            mode = 0  # single ended
+        else:
+            mode = 1  # differential
+
+        if (channel > 3 and mode == 1):
+            print('AIn: channel out of range for differential mode.')
```

Comment:
All `print()` should be changed to logging.error(), otherwise we'll never see them.